package.json

```diff
@@ -443,6 +443,11 @@
             ],
             "description": "%configuration.SAS.results.html.style%"
           },
+          "SAS.problems.log.enabled": {
+            "type": "boolean",
+            "default": false,
+            "description": "%configuration.SAS.problems.log.enabled%"
```

Comment:
It seems like a good idea to enable this by default? Not a deal breaker for getting this merged though.